### PR TITLE
feat(make): add linters target (#61)

### LINKED ISSUE #61: Include other go tools (`gofmt`, etc.) in the Makefile

### PR DESCRIPTION
**Pull request by moorereason**
Add linters target to run gofmt and golint checks against codebase.

Fixes #61

### CODE REVIEW

**Comment by xcoulon** [comment]
@moorereason thanks for this PR! I guess that it also would make sense to call the new `linters` goal when running on Travis-ci: https://github.com/bytesparadise/libasciidoc/blob/master/.travis.yml#L36 (not on AppVeyor because I'm pretty sure the command will fail). Is it something you could include in your PR, please?

**Comment by xcoulon** [comment]
Thanks for the PR update, @moorereason, but apparently, that does not work on [Travis-ci](https://travis-ci.org/bytesparadise/libasciidoc/jobs/442212246#L702): 
````
$ make linters
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
/bin/sh: 1: golint: not found
````
Also, I'm not sure if the linter will pass for all versions of Go defined in the build matrix, so we may want to only run the linter with the latest stable version of Go, for example.

**Comment by moorereason** [comment]
Pushed a new commit.  Feels a little hackish.  Do you want to maintain this?  I noticed after sending this PR that you have the GolangCI checker integrated into GH.  Is this PR necessary?  Open to suggestions

**Comment by xcoulon** [comment]
> Pushed a new commit. Feels a little hackish. Do you want to maintain this? I noticed after sending this PR that you have the GolangCI checker integrated into GH. Is this PR necessary? Open to suggestions

right, maybe providing a way to run [GolangCi linter](https://github.com/golangci/golangci-lint) on local would make more sense, indeed.

**Comment by moorereason** [comment]
That sounds like a good idea.  I forgot about GolangCI's cli interface.  I'll refactor.

File: Makefile [hunk 1]
```diff
@@ -58,12 +65,17 @@ help:/
           } \
         }' $(MAKEFILE_LIST)
 
-.PHONY: deps 
+.PHONY: install-devtools
+## Install development tools.
+install-devtools:
+	@go get -u -v $(DEVTOOLS)
```
Comment (xcoulon) [inline Makefile]: I didn't know it's possible to pass a list of libs to install! That's awesome!

**Pull request merged**